tests/test_audit_security_buffer.sh

```
@@ -39,7 +41,7 @@ assert_file_contains "knowledge_buffer_push 呼び出しがある" "$SKILL_FILE"
 echo ""
 echo "--- テスト3: CISO プロンプトに BUFFER_DIR 注入 ---"
 
-assert_file_contains "CISO プロンプトに BUFFER_DIR= が含まれる" "$SKILL_FILE" 'BUFFER_DIR=${BUFFER_DIR}'
+assert_file_contains "CISO プロンプトに BUFFER_DIR= が含まれる" "$CISO_PROMPT_FILE" 'BUFFER_DIR=${BUFFER_DIR}'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)